core/libp2p/multi/multistream.cpp

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "libp2p/multi/multistream.hpp"
+
+#include <iterator>
+#include <string_view>
+#include <algorithm>
+#include <spdlog/spdlog.h>
```

Comment:
I have just merged PR with a wrapper for logger, which is in `/core/common`. Please, use it to `createLogger(ClassName)` and write messages